superset/initialization/__init__.py

```diff
@@ -613,7 +613,11 @@ def __call__(
 
         # Talisman
         talisman_enabled = self.config["TALISMAN_ENABLED"]
-        talisman_config = self.config["TALISMAN_CONFIG"]
+        talisman_config = (
+            self.config["TALISMAN_DEV_CONFIG"]
+            if self.superset_app.debug
+            else self.config["TALISMAN_CONFIG"]
+        )
```

Comment:
I'm wondering if it's a good idea to have separate dev and non-dev configs. Should these maybe be `TALISMAN_DEFAULT_DEV_CONFIG` and `TALISMAN_DEFAULT_PROD_CONFIG`, and only if `TALISMAN_CONFIG` is undefined would we fall back to the default. Thoughts?

Reply:
We must have "unsafe-eval" in dev mode - React and Webpack use it and there's no way around it